cloud_automation/java/dataflow/src/main/java/org/datacommons/dataflow/CsvImport.java

```diff
@@ -0,0 +1,121 @@
+package org.datacommons.dataflow;
```

Comment:
Had the same question as Wei, this file seems to differ in some ways to https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/blob/master/java/dataflow-connector-examples/src/main/java/com/google/cloud/bigtable/dataflow/example/CsvImport.java. What are the differences?

Reply:
Per offline clarification, this sets up dataflow template, and needs to use a different BT API.